dev.mrquantumoff.mcmodpackmanager.yml

```diff
@@ -0,0 +1,32 @@
+# Flatpak Manifest
+app-id: dev.mrquantumoff.mcmodpackmanager
+runtime: org.gnome.Platform
+runtime-version: '42'
```

Comment:
43 is the latest. 42 will be eol in April/May

Reply:
I rewrote the app in Flutter/Dart, so now it's resolved (v6.0.0 uses freedesktop runtime)